### PR TITLE
chores: SWIFT v2 routing changes

### PR DESCRIPTION
**Pull request by nddq**
**Reason for Change**:

- Renaming NodeCIRDs to InfraVNETCIDRs
- Add route for virtual GW

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [x] adds unit tests
- [x] relevant PR labels added

**Notes**:

### CODE REVIEW

**Review by rbtr — DISMISSED** [review]
lgtm, @tyler-lloyd can you doublecheck?

File: cns/middlewares/mock/mockSWIFTv2.go [hunk 1]
```diff
@@ -130,20 +131,24 @@ func (m *SWIFTv2Middleware) SetRoutes(podIPInfo *cns.PodIpInfo) error {
 	podIPInfo.Routes = []cns.Route{}
 	switch podIPInfo.NICType {
 	case cns.DelegatedVMNIC:
+		virtualGWRoute := cns.Route{
+			IPAddress: virtualGW,
+		}
 		// default route via SWIFT v2 interface
 		route := cns.Route{
-			IPAddress: "0.0.0.0/0",
```
Comment (jaer-tsun) [inline cns/middlewares/mock/mockSWIFTv2.go:135]: we need to add the virtual gw route before we can add default via virtual gw

File: cns/middlewares/mock/mockSWIFTv2.go [hunk 2]
```diff
@@ -130,20 +131,24 @@ func (m *SWIFTv2Middleware) SetRoutes(podIPInfo *cns.PodIpInfo) error {
 	podIPInfo.Routes = []cns.Route{}
 	switch podIPInfo.NICType {
 	case cns.DelegatedVMNIC:
+		virtualGWRoute := cns.Route{
+			IPAddress: virtualGW,
```
Comment (jaer-tsun) [inline cns/middlewares/mock/mockSWIFTv2.go:135]: this route needs /32

File: cns/middlewares/mock/mockSWIFTv2.go [hunk 3]
```diff
@@ -130,20 +131,24 @@ func (m *SWIFTv2Middleware) SetRoutes(podIPInfo *cns.PodIpInfo) error {
...
+		}
 		// default route via SWIFT v2 interface
 		route := cns.Route{
-			IPAddress: "0.0.0.0/0",
+			IPAddress:        "0.0.0.0/0",
+			GatewayIPAddress: virtualGW,
```
Comment (jaer-tsun) [inline cns/middlewares/mock/mockSWIFTv2.go:140]: but this IP can't have /32 now :)

**Review by jaer-tsun — APPROVED** [review]

**Review by neaggarwMS — APPROVED** [review]

**Pull request merged**